packages/flutter_tools/flutter_analysis_options

```diff
@@ -11,6 +11,7 @@
 
 analyzer:
   language:
+    enableStrictCallChecks: true
```

Comment:
We should make sure to migrate changes to this file over to the engine's _embedder.yaml file.